src/renderer/components/+workspaces/workspace-menu.tsx

```diff
@@ -17,6 +20,16 @@ interface Props extends Partial<MenuProps> {
 export class WorkspaceMenu extends React.Component<Props> {
   @observable menuVisible = false;
 
+  activateWorkspace = (id: WorkspaceId) => {
+    const clusterId = workspaceStore.getById(id).lastActiveClusterId;
```

Comment:
What happens when there is no `lastActiveClusterId`?

Reply:
If `lastActiveCluster` id became `undefined` then user will be redirected to Landing page following code below.

Reply:
Maybe it's better to select first cluster in workspace and show it instead landing page? If no clusters -> landing?